openhtf/output/web_gui/__init__.py

```diff
@@ -223,7 +222,7 @@ def __getitem__(self, hostport):  # pylint:disable=invalid-name
     """Provide dictionary-like access to the station store."""
     if not isinstance(hostport, Hostport):
       raise ValueError('StationStore key must be a Hostport instance.')
-    return self.stations.setdefault(hostport, None)
+    return self.stations.get(hostport)
```

Comment:
Why was this a bug? It looks like the difference is that now, `self.stations` won't get a None for hostport's that weren't already in it. `setdefault()` does the same as a `get()`, but also writes the default back if it used the default